### DISCUSSION TITLE
Migrate from sqlite3 to pgsql or mysql

### DISCUSSION THREAD

**Discussion by anonkwe** [inventree/InvenTree#7190] [Q&A]
Hi folks,

Since we are using 2 separate instances of the app that are both linked to our actual sqlite database  (smh), we are in the urge to migrate our actual sqlite database to pgsql or mysql. I've already tried plenty of tools out there in order to migrate to mysql, but without complete success. 

Therefore, this is why I'm reaching out in here. What is the fastest and best way to achieve a migration from inventree's sqlite database to pgsql or mysql, no matter which one of those, as long as the database keeps its integrity? 

Any suggestion or queue is appreciated!

**Comment by SchrodingersGat** [comment]
You should just be able to follow the guide here - https://docs.inventree.org/en/stable/start/migrate/

Make sure you backup your data first!
